datasets/utils.py

```diff
@@ -0,0 +1,86 @@
+
+from enum import IntEnum
+import fnmatch
+import os
+from typing import List
+
+
+class DataTypes(IntEnum):
+    """Constants for data types."""
+    DATASET = 0
+    CHARSET = 1
+    PALETTE = 2
+    DICOMDIR = 3
+    JPEG = 4
+
+
+class Interface:
+    """Interface to pydicom-data.
```

Comment:
Depending on the other name discussion above, maybe something a little more specific for the class name: `DataInterface`, `DataStoreInterface`, even just `DataStore`, given any class defines an interface.

Reply:
Done, went with `DataStore`